Sprint-1/implement/dedupe.test.js

```diff
@@ -16,12 +16,25 @@ E.g. dedupe([1, 2, 1]) target output: [1, 2]
 // Given an empty array
 // When passed to the dedupe function
 // Then it should return an empty array
-test.todo("given an empty array, it returns an empty array");
+test("given an empty array, it returns an empty array", () => {
+  expect(dedupe([])).toEqual([]);
+});
 
 // Given an array with no duplicates
 // When passed to the dedupe function
 // Then it should return a copy of the original array
 
+test("given an array with no duplicates, it returns a copy of the original array", () => {
+  const input = [1, 2, 3, 4];
+  expect(dedupe(input)).toEqual(input);
+  expect(dedupe(input)).not.toBe(input); // different array instance
```

Comment:
We should also test that the returned array contains the same values as the input. Otherwise dedupe could return any array here and the test will still pass.

 ```suggestion
  expect(dedupe(input)).not.toBe(input); // different array instance
  expect(dedupe(input)).toEqual(input);
```